frontend/app/client-dashboard/page.js

```diff
@@ -1,11 +1,13 @@
+"use client";
 import Link from "next/link";
+import { useEffect, useState } from "react";
```

Comment:
why this `useEffect` is here? I don't see it being used anywhere. if you don't need it you can remove `use client` also

Reply:
I think i forgot to clean it